cfy_manager/components/stage/stage.py

```diff
@@ -193,7 +202,7 @@ def remove(self):
     def start(self):
         logger.notice('Starting Stage...')
         self._run_db_migrate()
-        service.restart(STAGE)
+        service.start(STAGE)
```

Comment:
Same query as for the service.restart -> service.start in composer, above.

Reply:
will check it